backbone.js

```diff
@@ -979,14 +979,14 @@
     sort: function(options) {
       if (!this.comparator) throw new Error('Cannot sort a set without a comparator');
       options || (options = {});
+      var comparator = _.isFunction(this.comparator) ? _.bind(this.comparator, this) : this.comparator;
```

Comment:
Yap, [there's another place](https://github.com/jashkenas/backbone/blob/1.2.1/backbone.js#L440) that relies on `iteratee(..., this)` but in that case I can advise users to just provide their callbacks pre-bound.